lib/features/login/presentation/widgets/login_message_widget.dart

```diff
@@ -54,6 +55,10 @@ class LoginMessageWidget extends StatelessWidget {
               } else if (failure is GetTokenOIDCFailure && failure.exception is NoSuitableBrowserForOIDCException) {
                 return AppLocalizations.of(context).noSuitableBrowserForOIDC;
               } else if (failure is FeatureFailure) {
+                if (failure.exception is NoNetworkError) {
+                  return AppLocalizations.of(context).youAreOffline;
+                }
+
                 return _toastManager?.getMessageByException(context, failure.exception)
                   ?? AppLocalizations.of(context).unknownError;
```

Comment:
```suggestion
              } else if (failure is FeatureFailure && failure.exception is NoNetworkError) {
```

to avoid nesting `if/else`.

Reply:
Done